app/src/main/kotlin/com/x8bit/bitwarden/data/auth/repository/model/Organization.kt

```diff
@@ -21,4 +23,5 @@ data class Organization(
     val role: OrganizationType,
     val keyConnectorUrl: String?,
     val userIsClaimedByOrganization: Boolean,
+    val limitItemDeletion: Boolean = false,
```

Comment:
⛏️ We need to declare a default for `limitItemDeletion` since we're setting it explicitly in the extension function. The default only needs to be declared on the JSON model for backwards compatibility.

Reply:
There is a reason why I added the default here, this change is just a temporary fix for self-hosted environment until we remove the feature flag and the logic gets deleted from the codebase. 
Without it I need to add around 80 changes in the test files, didn't seem to be worth it.